content/docs/studio/user-guide/model-registry/use-models.md

```diff
@@ -59,23 +59,29 @@ credentials set up in your environment.
 
 ## Deploying and publishing models in CI/CD
 
-A popular deployment option is to use CI/CD pipelines triggered by new Git tags
-to publish or deploy a new model version. Since GTO registers versions and
-assigns stages by creating Git tags, you can set up a CI/CD pipeline to be
+A popular deployment option is to **use CI/CD pipelines triggered by new Git
+tags to publish or deploy a new model version**. Since GTO registers versions
+and assigns stages by creating Git tags, you can set up a CI/CD pipeline to be
 triggered when the tags are pushed to the repository.
 
-To see an actual example, check out
+**To see an example**, check out
 [the workflow in the `example-gto` repository](https://github.com/iterative/example-gto/blob/main/.github/workflows/gto-act-on-tags.yml).
 This workflow uses
 [the GTO GitHub Action](https://github.com/iterative/gto-action) that interprets
 a Git tag to find out the model's version and stage assignment (if any), reads
 annotation details such as `path`, `type` and `description`, and downloads the
 model binaries if needed.
 
-The `example-gto` repository also includes an
-[example workflow that deploys a model](https://github.com/iterative/example-gto/blob/mlem/.github/workflows/deploy-model-with-mlem.yml)
-from CI/CD using MLEM.
-
-Finally, you can find examples of building a Docker image with a model and
-deploying it to the cloud in the
+For help **building an end-to-flow from model training to deployment using the
+DVC model registry**, refer the
+[tutorial on automating model deployment](https://iterative.ai/blog/sagemaker-model-deployment).
+This tutorial adapts the above example workflow to get the details of the
+requested deployment. It then uses the `dvc get --show-url` command to get the
+model file path. Finally, it invokes a custom deployment script that deploys the
+model to **Amazon SageMaker**.
+[Here](https://github.com/iterative/example-get-started-experiments/blob/main/.github/workflows/deploy-model.yml)
+is the complete workflow script.
```

Comment:
```suggestion
[tutorial on automating model deployment to Sagemaker](https://iterative.ai/blog/sagemaker-model-deployment).
[Here](https://github.com/iterative/example-get-started-experiments/blob/main/.github/workflows/deploy-model.yml)
is the complete workflow script.
```

I'd suggest to remove extra details here, they're too low-level (and hard to understand without reading the blog post anyway). If someone is interested, he can check out blog post or example script.